server/src/server/command.c

```diff
@@ -130,10 +131,27 @@ char		*leave(t_game_manager **manager, char *identity, char *optional)
 char		*forward(t_game_manager **manager, char *identity, char *optional)
 {
   char		log[50];
-  //  t_player	*player;
+  t_player	*player;
 
-  if ((*manager)->ready)
+  if ((*manager)->ready) {    
```

Comment:
ah oui pas bête, c'est vrai que si la partie se termine sur le joueur précédent, le jeu sera terminé et il est probable de devoir testé ainsi sur toute les fonctions in game, bien vu!